components/brave_wallet_ui/page/screens/fund-wallet/deposit-funds.tsx

```diff
@@ -321,10 +316,10 @@ export const DepositFundsScreen = () => {
           ) &&
             <StepsNavigation
               goBack={onBack}
-              onSkip={goToPortfolio}
               skipButtonText={getLocale('braveWalletButtonDone')}
```

Comment:
Do we need this text if the skip button functionality was removed?

Reply:
Removed!